contracts/BondController.sol

```diff
@@ -268,6 +270,7 @@ contract BondController is IBondController, OwnableUpgradeable {
      */
     function getTrancheLetter(uint256 index, uint256 _trancheCount) internal pure returns (string memory) {
         bytes memory trancheLetters = bytes("ABCDEFGHIJKLMNOPQRSTUVWXY");
+        require(index < trancheLetters.length, "BondController: invalid tranche letter");
```

Comment:
It checks index against trancheLetters.length, but trancheLetters excludes Z so when there are 26 tranche ratios this fails despite there being enough letters for each tranche.
People shouldn't really be messing about with 25 or 26 tranche bonds anyway, really, but from a "correctness" point of view we should correct this I think. Cheapest way gas-wise I think would be to add +1 to the condition:
`require(index < (trancheLetters.length + 1), "BondController: invalid tranche letter");`